apps/passport-server/.env.local.example

```diff
@@ -73,5 +73,15 @@ SERVER_EDDSA_PRIVATE_KEY=
 TELEGRAM_BOT_TOKEN=
 TELEGRAM_BOT_START_DELAY_MS=0
 
-# Disable Pretix sync for local development
-PRETIX_SYNC_DISABLED=TRUE
+# Disable Pretix sync for local development. 
+# Now that we use Pretix sync to load the dev event + tickets, we want to keep Pretix sync enabled.
```

Comment:
I think we'd rather just have a more general comment here. This makes sense to current members of our team, but not for someone new onboarding to the codebase

```suggestion
# To disable syncing tickets from Pretix for local development
```